nextcord/ui/modal.py

```diff
@@ -164,7 +164,7 @@ def _expires_at(self) -> Optional[float]:
         return self.__timeout_expiry
 
     def add_item(self, item: Item) -> Modal:
-        """Adds an item to the modal.
+        r"""Adds an item to the modal.
```

Comment:
Whats this for?

Reply:
That's not my change, the pre-commit bot added it automatically, as seen in the commits tab.

Reply:
Interesting